### DISCUSSION TITLE
Upgrade to Pro from free

### DISCUSSION THREAD

**Discussion by Barbas70** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1386] [General]
Good morning,

I just bought aldente Pro.

I'm currently using the free version.

I wanted to know if to switch to the pro version it is enough to "activate license" in the free version or if it is better to uninstall (and how) the free version and install the pro one (the files downloaded for the free version and the pro look different)

Thank you

Robert

**Comment by MatthiasKerbl** [comment]
Hi Robert,

You can just click "activate license" within AlDente Free to unlock the Pro features. No need to download any other files.

Best,
Matthias

**Reply by Barbas70** [reply]
thank you
